tlatools/org.lamport.tlatools/src/pcal/trans.java

```diff
@@ -1295,8 +1294,7 @@ private static void WriteStringVectorToFile(final List<String> inputVec, String
     * each element of inputVec written on a new line.                      *
     ***********************************************************************/
     {
-    	// TODO use Apache Commons for this
-        try (final BufferedWriter fileW = new BufferedWriter(new FileWriter(fileName))) {
+        try (final BufferedWriter fileW = Files.newBufferedWriter(Path.of(fileName))) {
```

Comment:
Please move the changes related to java.nio here and below to a separate commit of this PR, i.e., a commit for the refactoring to java.nio.  If we ever have to bisect because java.nio changes in subtle ways, it will be easier with more fine-grained commits.